### PR TITLE
Database log streamlined

### PR DESCRIPTION
**Pull request by vivek42537**
#### Ticket Link

Fixes

#### Release Note

```release-note

```

### CODE REVIEW

File: cmd/mirroring/logInfo.txt [hunk 1]
```diff
@@ -0,0 +1,32 @@
+1528952 e0a8822e61ba905c6efb216666c3e09f37f403c5d1f55e8cef8d964ddddefabf
```
Comment (SantiagoTorres) [inline cmd/mirroring/logInfo.txt:1]: can you remove tihs file from the PR?

Comment (SantiagoTorres) [reply cmd/mirroring/logInfo.txt:1]: and the .DS_Store's above. These are apple thumbnails that shouldn't be checked into the repo

File: cmd/mirroring/main.go [hunk 2]
```diff
@@ -37,6 +41,49 @@ const (
...
+
+type Signature struct {
+	Content   string    `json:"content"`
+	Format    string    `json:"format"`
+	PublicKey PublicKey `json:"publicKey"`
+}
+
+type PublicKey struct {
+	Content string `json:"content"`
+}
```
Comment (SantiagoTorres) [inline cmd/mirroring/main.go:85]: Not sure if we need the structs above, do we?

File: cmd/mirroring/main.go [hunk 3]
```diff
@@ -163,6 +210,53 @@ func main() {
...
+
+		log.Println("currentLastID: ", id, "newTreeSize: ", newTreeSize)
+		if newTreeSize-id != 0 { //last id in our database compared to new tree size
+			// mirroring.rows, err = mirroring.getLatestX(database, (newTreeSize - id))
+			for i := id + 1 ; i < newTreeSize; i++ {
+				_, payload, _ := mirroring.GetLogEntryByIndex(i, rekorClient)
+				// log.Println("payload value: %s", payload.Body.(string))
+				// b, _ := base64.StdEncoding.DecodeString(payload.Body.(string))
+				pay, _ := payload.MarshalBinary()
+				// b, _ := base64.StdEncoding.DecodeString(pay)
```
Comment (SantiagoTorres) [inline cmd/mirroring/main.go:227]: can you remove the commented code in this block?

File: go.mod [hunk 4]
```diff
@@ -5,6 +5,7 @@ go 1.16
 require (
 	github.com/go-openapi/runtime v0.19.30
 	github.com/google/trillian v1.3.14-0.20210713114448-df474653733c
+	github.com/mattn/go-sqlite3 v1.14.11 // indirect
```
Comment (SantiagoTorres) [inline go.mod:8]: we probably don't need any of the *.db's above, can you remove them?

File: mirroring/database.go [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+	_ "github.com/mattn/go-sqlite3"
+)
+
+type Data struct {
+	ID      int64
+	Payload string
+}
+
+func initTable(database *sql.DB) error {
+	statement, err := database.Exec("CREATE TABLE entries (idx INTEGER PRIMARY KEY NOT NULL, payload TEXT)") //sql query string (if no db)
```
Comment (SantiagoTorres) [inline mirroring/database.go:18]: can you move the comment to a separate line?

File: mirroring/database_test.go [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+	// if(id != 1999){
+	// 	t.Errorf("Expected Result 1999, instead retrieved %d", id)
+	// 	t.Errorf("payload is: %s", stringPay)
+	// }
+	var id int
+    var payload string
+    for rows.Next() {
+        rows.Scan(&id, &payload)
+        fmt.Println(strconv.Itoa(id) + ": " + payload)
+    }
```
Comment (SantiagoTorres) [inline mirroring/database_test.go:49]: we probably need to run gofmt on this file.

**Review by SantiagoTorres — CHANGES_REQUESTED** [review]
Happy to see progress! I left a couple of comments. Let's start with removing the unneeded files first!

**Comment by haydentherapper** [comment]
Closing older PRs, feel free to reopen if you want to continue this work.

**Pull request closed without merging**